helm/hpcc/values.yaml

```diff
@@ -534,6 +534,23 @@ esp:
   #resources:
   #  cpu: "250m"
   #  memory: "1G"
+- name: ldapenvironment
+  #ldapenvironment is a stand alone ESP service used to help stand up new HPCC LDAP Environments
+  application: ldapenvironment
+  auth: ldap
+  #specify the hpccRootName, without leading "ou="
+  hpccRootName: hpcc,dc=myldap,dc=com
```

Comment:
Again, curious why hpccRootName doesn't already start with ou=?

Reply:
LDAPSecurityManager anomaly, see proposal above to allow admin to optionally specify it

Reply:
ok, if it's an expected format from the secmgr.

Reply:
I changed the code to allow the more familiar "ou=rootName" syntax